### DISCUSSION TITLE
视频下载功能增加对festival页面的支持

### DISCUSSION THREAD

**Discussion by Hacinsl** [the1812/Bilibili-Evolved#2999] [功能建议 / Ideas]
在部分活动页面，功能栏不提供下载按钮，希望增加

**Comment by Geeyun-JY3** [comment]
[2.1.3](https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.3) 不支持吗

**Reply by Hacinsl** [reply]
似乎只支持了B站拜年纪，一些活动页面仍然不显示下载按钮
![image](https://user-images.githubusercontent.com/63764883/153755057-b97c4629-666a-4722-8920-695f65fdfba6.png)
https://www.bilibili.com/festival/VSF2022live?bvid=BV1UT4y1k7Zn&p=1&t=0
版本v2.1.3-4-g332b4c325

**Reply by the1812** [reply]
是的, 其他拜年祭页面还没加上
